client/src/components/ui/Select.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+    export let options: Array<string> = [];
```

Comment:
It's a little bit more idiomatic this way. 👍 
```suggestion
    export let options: string[] = [];
```